statistics-reports.md

```diff
@@ -0,0 +1,202 @@
+# Displaying usage statistics
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Statistics for a DSpaceObject
+**GET /api/statistics/usage-reports**
+
+As we don't have a use case to iterate over all the existing usage reports, the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error-codes).
+
+## Single statistic
+**GET /api/statistics/usage-reports/<:id>**
+
+This endpoint provides a specific statistic
+
+An example JSON response document to `/api/statistics/usage-reports/1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries`:
+```json
+{
+    "id": "1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries",
+    "type": "usage-report",
+    "report-type": "TotalVisits",
+    "points": [
+        {
+            "label": "United States",
+            "type": "country",
+            "id": "US",
+            "values": {
+                "views": 2
+            }
```

Comment:
can you describe such attributes and their meaning? which are the possible values for type?

Reply:
I've included the possible values for type.
I've also adjusted the example for the date because the "Total Visits Per Month" graph contains multiple values per row in the table (instead of a row per month)